### DISCUSSION TITLE
EPUB reader

### DISCUSSION THREAD

**Discussion by heyhippari** [jellyfin/jellyfin-vue#150] [Features]

**Comment by Bitwolfies** [comment]
Not sure if this fits here or in server, but Ill ask here as to not make a issue. Is a richer epub reader planned? With notes, bookmarks, highlights, a table of consents for all of it, ect. I've seen the nextcloud reader thrown around a lot, as it can do those things. Thank you!

**Reply by ThibaultNocchi** [reply]
For this to be possible, I think server will have to support it
